### ISSUE TITLE
handle case with empty `xspectra`  for WV

### ISSUE THREAD

**Issue by agrouaze** [umr-lops/xsar_slc#99]
issue already handled for IW.

```python
raceback (most recent call last):
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/bin/L1B_xspectra_WV_SLC_IFR", line 8, in <module>
    sys.exit(main())
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xsarslc/scripts/L1B_xspectra_WV_SLC_IFR.py", line 104, in main
    generate_WV_L1Bxspec_product(slc_wv_path=slc_wv_path, output_filename=output_filename,
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xsarslc/scripts/L1B_xspectra_WV_SLC_IFR.py", line 144, in generate_WV_L1Bxspec_pr
oduct
    xs0 = proc.compute_WV_intraburst_xspectra(dt=dt,
  File "/home/datawork-cersat-public/cache/project/mpc-sentinel1/workspace/mamba/envs/xsar_oct22/lib/python3.9/site-packages/xsarslc/processing/xspectra.py", line 114, in compute_WV_intraburst_xspectra
    xspectra = xspectra.drop(
AttributeError: 'NoneType' object has no attribute 'drop'
INFO:prun:[2023-03-27T13:40:55Z] End of processing for input line 49 : '/home/datawork-cersat-public/project/mpc-sentinel1/data/esa/sentinel-1a/L1/WV/S1A_WV_SLC__1S/2016/297/S1A_WV_SLC__1SSV_20161023T125412_20161023T131401_013617_015D1B_B7A8.SAFE/measurement/s1a-wv1-slc-vv-20161023t131314-20161023t131317-013617-015d1b-079.tiff tiles20km 1.4b /home1/scratch/agrouaze/l1b/wv/'. EXIT_CODE=1
```

**Comment by agrouaze** [comment]
this issue has not been found so far because for the few WV tests we did, there was no land-mask.

**Comment by lanougue** [comment]
solved in #103
